### PR TITLE
Add feature to change lw and color

### PR DESCRIPTION
**Pull request by erica-pathos**

### CODE REVIEW

File: starbars/__init__.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ def draw_annotation(
     top_margin=0.05,
     text_distance=0.02,
     fontsize=10,
+    lw=1, c='k', **kwargs
```
Comment (elide-b) [inline starbars/__init__.py:22]:
```suggestion
    line_width=1, 
    color='k', 
    **kwargs
```

File: starbars/__init__.py [hunk 2]
```diff
@@ -73,7 +74,7 @@ def draw_annotation(
 
     # Draw the statistical annotation
     for bar, text, (_, _, pvalue) in zip(bars, texts, annotations):
-        ax.plot([c[0] for c in bar], [c[1] for c in bar], lw=1.5, c="k")
+        ax.plot([c[0] for c in bar], [c[1] for c in bar], lw=lw, c=c, **kwargs)
```
Comment (elide-b) [inline starbars/__init__.py:77]:
```suggestion
        ax.plot([c[0] for c in bar], [c[1] for c in bar], lw=line_width, c=color, **kwargs)
```

File: starbars/__init__.py [hunk 3]
```diff
@@ -82,7 +83,7 @@ def draw_annotation(
                 ha="center",
                 va="center",
                 fontsize=fontsize,
-                color="k",
+                color=c,
```
Comment (elide-b) [inline starbars/__init__.py:86]:
```suggestion
                color=color,
```

**Review by elide-b — CHANGES_REQUESTED** [review]
Hello! Thank you for your great contribution :)
I put some suggestions to have the style more in line with the current one. As soon as you accept them I can merge and release it ✨

**Review by erica-pathos — COMMENTED** [review]
Yea, it looks good, thanks.

**Pull request closed without merging**